hillary25/rain_or_shine
# download from production theme

### Goal
Initial theme download.

### Testing Instructions
- Visit [Homepage](https://dha8671p4u4fg82p-31874285700.shopifypreview.com). Make sure site builds without error.
- Ensure that all files from the production theme have been downloaded.

### Code Request Quality Checklist
- [x] Code is escaped and localized appropriately.   

### Reviewers
@reviewer1  
@reviewer2
